.github/workflows/publish.yml

```diff
@@ -46,6 +46,7 @@ jobs:
           fromJSON(steps.inputs.outputs.result).repo == 'sentry-migr8' && fromJSON(steps.inputs.outputs.result).merge_target == 'tmp-merge-target' ||
           fromJSON(steps.inputs.outputs.result).repo == 'sentry-javascript' && fromJSON(steps.inputs.outputs.result).merge_target == 'v7' ||
           fromJSON(steps.inputs.outputs.result).repo == 'sentry-javascript' && fromJSON(steps.inputs.outputs.result).merge_target == 'master' ||
+          fromJSON(steps.inputs.outputs.result).repo == 'sentry-wizard' && fromJSON(steps.inputs.outputs.result).merge_target == '1.x' ||
```

Comment:
this branch does not have branch protection

Reply:
Stephanie just updated protection rules for 1.x 🙏

Reply:
the protection rule does not require CI to pass -- only approval.  should it have some sort of automated quality checks?

Reply:
It seems it [there is](https://github.com/getsentry/sentry-wizard/blob/1.x/.github/workflows/build.yml)/was some CI for 1.x (this is old, we're on 3.x right now) but I'm not sure why CI isn't running for 1.x at the moment. It should at least run for a PR but apparently [it doesn't](https://github.com/getsentry/sentry-wizard/pull/584). Any ideas?

I just want to get a patch release out because a user still relying on v1 contributed a small fix. I think we already achieved branch protection in the sense that we require approval before merging. Is this enough?

Reply:
actually taking a step back -- seems like we're going through a ton of work for something we don't support.  should we just remove the docs showing `@1` and claim this is unsupported territory?

Reply:
I would ask that we focus on the important aspects here which is to enable obtaining the craft config of v1 for v1 releases instead of the recent v3 one. The configs diverged. For instance we added the `brew` target in the meantime and I definitely don’t want to release 1.x to `brew`. The alternative would be to hack around in the v3 craft config temporarily or get one of the ~4 people in the company with the necessary permissions to uncheck the irrelevant check boxes in the publish issue but should we ever need to do another v1 release, we’re right back where we started.

> should we just remove the docs showing @1 and claim this is unsupported territory?

I’m trusting the Mobile SDK team that v1 is mentioned for good reason in docs. As we can see, users do rely on it, AFAICT for older RN project setups where the v3 wizard wouldn’t work anymore. Furthermore, again, a user is relying on v1 and they contributed the fix themselves. Releasing should be the easy part here.

> seems like we’re going through a ton of work

Frankly, most of what’s necessary for this was already done and took me ~30 minutes. Using the correct craft config is last piece of the puzzle and it’s by now taking more time than everything else combined :)

I do agree, this is not a p0 change but again, let’s just get this out to improve DX for our users and avoid spending more time on this than necessary.